docs/getting-started/aws-1click.md

```diff
@@ -0,0 +1,270 @@
+| _Auto-assign Public IP_ | Enable                                                                                                           |
+| _Other fields_          | Follow default setting                                                                                           |
+
+### Step 2-4. Add Storage
+- Follow default setting with `125` Throughput (MB/s)
+
+### Step 2-5. Add Tags
+- Create Tag if needed
+  
+### Step 2-6. Configure Security Group
+By default, the image create a new security group for you allowing ssh, p2p, rpc, and rest ports.
+You can remove the default opened rules or add more rules for example: grpc(9090), node_exporter(9100), prometheus (26660), etc.
+| Default Port            | Description                                                                                                      |
+| ----------------------- | ---------------------------------------------------------------------------------------------------------------- |
+| _22_                    | ssh                                                                                                              |
+| _26656_                 | p2p                                                                                                              |
+| _26657_                 | rpc                                                                                                              |
+| _1317_                  | rest                                                                                                             |
+You should consider updating your security group rules to allow access from known IP addresses only.
+
+### Step 2-7. Review Instance Launch
+Click "Launch" once you finish the review
+![](./assets/aws_launch.png)
+
+You can either create a new key pair or import a key pair. Follow this [instruction](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html#prepare-key-pair)
+
+### Step 2-8. Connect to your instance
+
+Once the deployment is completed, you can connect to your instance via SSH.
+```bash
+$ ssh ubuntu@PUBLIC_IP -i YOUR_KEY.pem
+```
+Afterwards, go to the `chain` directory, and you will find all the essential binaries for setting up your node:
+
+```bash
+$ cd /chain/
+$ ll
+total 32
+drwxr-xr-x  7 crypto crypto 4096 Mar 20 14:51 ./
+drwxr-xr-x 20 root   root   4096 Mar 25 09:46 ../
+drwxr-xr-x  4 crypto crypto 4096 Mar 20 14:47 .chain-maind/
+drwxr-xr-x  2 crypto crypto 4096 Mar 20 14:44 bin/
+drwxr-xr-x  2   3434   3434 4096 Mar  5 09:41 node_exporter-1.1.2.linux-amd64/
+-rwxr-xr-x  1 crypto crypto  535 Mar 20 14:43 reconfig.sh*
+drwxr-xr-x  2 root   root   4096 Mar 20 14:43 release_binary/
+drwxr-xr-x  2 crypto crypto 4096 Mar 20 14:47 sockets/
+```
+
+
+## Step 3. Running a full node
+
+### Step 3-1. Use the reconfiguration script
+
+Now we use the reconfiguration script `reconfig.sh` to choose the existing networks to join.
+
```

Comment:
ask users to run `"sudo -u crypto ./reconfig.sh"`?

Reply:
can do